pkg/bloomgateway/metrics.go

```diff
@@ -66,6 +67,13 @@ func newServerMetrics(registerer prometheus.Registerer, namespace, subsystem str
 			Help:      "Total amount of chunk refs filtered by bloom-gateway",
 			Buckets:   prometheus.ExponentialBucketsRange(1, 100e3, 10),
 		}),
+		receivedFilters: promauto.With(registerer).NewHistogram(prometheus.HistogramOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "request_filters",
+			Help:      "Number of filters per request.",
+			Buckets:   prometheus.LinearBuckets(0, 1, 20),
```

Comment:
I don't think we'll need 20 1-width buckets. I've suggested an exponential increasing range up to 256
```suggestion
			// 1->256
			Buckets:   prometheus.ExponentialBuckets(1, 2, 8),
```

Reply:
Done